napari/layers/image/image.py

```diff
@@ -187,7 +189,9 @@ class _ImageBase(IntensityVisualizationMixin, Layer):
         Gamma correction for determining colormap linearity.
     interpolation : str
         Interpolation mode used by vispy. Must be one of our supported
-        modes.
+        modes. 'custom' is a special value which uses 'linear' interpolation
+        to sample the texture with a custom convoution kernel (provided with
+        'custom_interpolation_kernel_2d') around the pixel.
```

Comment:
almost! not quite clear enough I think, how about...

'custom' is a special mode for 2D interpolation in which a regular grid of samples are taken from the texture around a position using 'linear' interpolation before being multiplied with a custom interpolation kernel (provided with 'custom_interpolation_kernel_2d').

Something else that's a little unclear to me is where the center of the kernel is if the kernel has even sidelength, do. you know what happens there? :)

Reply:
The center of the kernel is always centered, so the exact center is not sampled.